behaviour/__init__.py

```diff
@@ -5,5 +5,3 @@
 tryhard = behaviour.tryhard.TryHardBehaviour()
 dealer = behaviour.dealer.DealerBehaviour()
 human = behaviour.human.HumanBehaviour()
-
-__all__ = [tryhard, human, dealer]
```

Comment:
Would be nice to use the same approach as command so there is no need to add new behaviours in this file.